app/test.jsx

```diff
@@ -405,6 +414,10 @@ class App extends Component {
                     </SmartChart>
                 </div>
                 <div className="action-section">
+                    <div className="form-row">
+                        Widget <br />
```

Comment:
Instead of breaking, I think you should use a `<p>` tag instead, looks cleaner that way.

Reply:
this is our `test` file, so mostly we try to handle it very simple